Terminal.Gui/Views/Menu/ContextMenu.cs

```diff
@@ -188,13 +188,14 @@ public void Show ()
 			position.Y = 0;
 		}
 
-		_menuBar = new MenuBar (new [] { MenuItems }) {
+		_menuBar = new MenuBar {
 			X = position.X,
 			Y = position.Y,
 			Width = 0,
 			Height = 0,
 			UseSubMenusSingleFrame = UseSubMenusSingleFrame,
-			Key = Key
+			Key = Key,
+			Menus = [MenuItems]
```

Comment:
Woo!

Glad to see collection expressions getting used. 😀

Reply:
At least I'm use them on the changes I really have to do but not all yet.

Reply:
Haha hey I'm just thrilled to see any use of them. They can have a surprisingly large impact because of how they designed that feature.